api/src/pdc/services/export/models/CarpoolRow.ts

```
@@ -21,48 +21,52 @@
+export class CarpoolRow<T extends { [k: string]: unknown }> {
+  constructor(protected data: T) {}
 
   /**
    * pick fields or return the whole data
    *
    * @param fields
    */
-  public get(fields?: string[]): Partial<CarpoolRowData> {
-    return fields && fields.length ? pick(this.data, fields as any) : this.data;
+  public get(fields?: string[]): Partial<T> {
+    return fields && fields.length ? pick(this.data, fields) : this.data;
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)